vmdb/app/controllers/miq_ae_customization_controller/dialogs.rb

```diff
@@ -192,12 +192,13 @@ def dialog_edit
     assert_privileges("dialog_edit")
     case params[:button]
     when 'cancel'
+      dialog_label = session[:edit][:current][:label] if params[:id]
```

Comment:
Why not put the dialog_label assignment inside the `if` statement a few lines below?